tables/intro-table/step5.md

```diff
@@ -1,4 +1,4 @@
-In this step, we'll move our row/column definitions to an external file and add more stub data to make the next part of the lesson more interesting. This will also help keep our component file simple and readable.
+In this step, we'll move our row/column definitions to an external file and add three more rows of stub data to make the next part of the lesson more interesting.
```

Comment:
I think you mean `sub` data

Reply:
I was generally referring to the table data as "stub" data, as it's only used for illustration purposes. Is "stub" confusing? Maybe "sample" would be better?

Reply:
Oh! Yeah I didn't understand what "stub" data is so that would be great to use sample or another more familiar term